### PR TITLE
Clarified TLSv1.3 support

### PR DESCRIPTION
**Pull request by Night1**
Clarified that TLS1.2 is not the most recent versions and that TLSv1.3 is not suppored.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/technical-reference-details-about-encryption.md [hunk 1]
```diff
@@ -76,7 +78,7 @@ As of June 2016, Office 365 no longer accepts a SHA-1 certificate for outbound o
...
 
 > [!IMPORTANT]
-> Be aware that TLS versions deprecate, and that deprecated versions *should not be used* where newer versions are available. In other words, anywhere where it's listed that TLS 1.0, 1.1, and 1.2 are supported, choose the *most recent* version (TLS 1.2).
+> Be aware that TLS versions deprecate, and that deprecated versions *should not be used* where newer versions are available. In other words, anywhere where it's listed that TLS 1.0, 1.1, and 1.2 are supported, choose the *most recent* supported version (TLS 1.2), TLS v1.3 is currently not supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/technical-reference-details-about-encryption.md]:
```suggestion
> Be aware that TLS versions deprecate, and that deprecated versions *should not be used* where newer versions are available. In other words, anywhere where it's listed that TLS 1.0, 1.1, and 1.2 are supported, choose the *most recent* supported version (TLS 1.2). TLS 1.3 is currently not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edits.

File: microsoft-365/compliance/technical-reference-details-about-encryption.md (hunk 1 above)
Comment (Night1) [reply microsoft-365/compliance/technical-reference-details-about-encryption.md]: Agreed, 1.0 and 1.1 must not be used for PCI3.1  compliance neither for according to NIST for government connection (fedramp) same goes within most of NATO NSAs too.  Not to mention HIPAA  (NIST Special Publication 800-52r2) requiring TLSv1.2 and  TLSv1.3 by January 1st '24.

Comment (mypil) [reply microsoft-365/compliance/technical-reference-details-about-encryption.md]: @Night1 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by mypil — CHANGES_REQUESTED** [review]

File: microsoft-365/compliance/technical-reference-details-about-encryption.md (hunk 1 above)
Comment (mypil) [inline microsoft-365/compliance/technical-reference-details-about-encryption.md]:
```suggestion
> Be aware that TLS versions deprecate, and that deprecated versions *should not be used* where newer versions are available. In other words, anywhere where it's listed that TLS 1.0, 1.1, and 1.2 are supported, choose the *most recent* supported version (TLS 1.2), TLS 1.3 is currently not supported.
```

Comment (mypil) [reply microsoft-365/compliance/technical-reference-details-about-encryption.md]: @Night1 - please merge this commit similar to what @JohanFreelancer9 suggested. Please use TLS 1.3 in your statement.

Comment (nenonix) [inline microsoft-365/compliance/technical-reference-details-about-encryption.md]:
```suggestion
> Be aware that TLS versions deprecate, and that deprecated versions *should not be used* where newer versions are available. In other words, anywhere where it's listed that TLS 1.0, 1.1, and 1.2 are supported, choose the *most recent* supported version (TLS 1.2). TLS 1.3 is currently not supported.
```

**Review by nenonix — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review.

Thank you.

cc: @kenwith

**Comment by Night1** [comment]
any update on this?

**Comment by mypil** [comment]
@KCCross - We would like to follow up on this. Please assist to review and merge this PR. Thank you.

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Do you have any updates on this? This will help fix issue #980

There is also a duplicate PR #1001 which I closed.

@LauraWi - Would you be able to help merge this PR?

Thank you.

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Can you please review and merge this PR?

Thank you.

**Comment by Night1** [comment]
This seams to be a hard one 💯 @mypil any updates?

**Comment by GiantCrocodile** [comment]
Having this updated would be quite nice after seeing it's from 2019-11 and caused other issues to be created.

**Pull request merged**